presto-native-execution/presto_cpp/presto_protocol/connector/clp/presto_protocol_clp.yml

```
@@ -0,0 +1,39 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+AbstractClasses:
+  ColumnHandle:
+    super: JsonEncodedSubclass
+    comparable: true
+    subclasses:
+      - { name: ClpColumnHandle,         key: clp }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)